source/common/ssl/context_impl.cc

```diff
@@ -91,12 +92,16 @@ ContextImpl::ContextImpl(Stats::Scope& scope, const ContextConfig& config)
       }
       if (item->crl) {
         X509_STORE_add_crl(store, item->crl);
+        has_crl = true;
       }
     }
     if (ca_cert_ == nullptr) {
       throw EnvoyException(fmt::format("Failed to load trusted CA certificates from {}",
                                        config.certificateValidationContext()->caCertPath()));
     }
+    if (has_crl) {
```

Comment:
should this cover certificateRevocationList() part below?

Reply:
Nope, those flags are set when `crl` field is configured separately.